eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -56,6 +60,15 @@ jobs:
           filePath: "tools/AnalyzeDeps/AnalyzeDeps.ps1"
           arguments: '-PackagesPath "$(Pipeline.Workspace)/packages/*.nupkg" -LockfilePath "$(Build.SourcesDirectory)/eng/Packages.Data.props" -OutPath "$(Build.ArtifactStagingDirectory)/dependencies.html"'
           pwsh: true
+      - pwsh: |
+          $ServiceDir = If ( '${{parameters.ServiceDirectory}}' -eq '*') {'sdk'} Else { '${{parameters.ServiceDirectory}}' }
```

Comment:
We will want to put this script in a shared location similar to the VerifyOSScript so we can use it in all our pipelines.